README.md

```diff
@@ -1,5 +1,5 @@
 # ModelicaSpecification
-This repository contains the Modelica Language Specification, hosted at https://github.com/modelica/ModelicaSpecification.
+This repository contains the Modelica Language Specification, hosted at https://github.com/modelica/ModelicaSpecification. Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects).
```

Comment:
(I always thought it was capitalized _MAP-Lang_, but all upper case it at least consistent with the linked list of projects.)